From c84aa5a03eaa44fd8b0f3ad2132c0295fc31a73c Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Tue, 8 Dec 2020 17:11:11 +0530
Subject: [PATCH 5/5] Fix Wifi Enable issue on R on top of below change

"GTS failure fix for data_between_core_and_vendor_violators"

Change-Id: Ifd657467d22ccb4b92565eea27d22516c6420946
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/graphics/mesa/vendor_init.te b/graphics/mesa/vendor_init.te
index cc5eba7..23389fa 100644
--- a/graphics/mesa/vendor_init.te
+++ b/graphics/mesa/vendor_init.te
@@ -1,6 +1,6 @@
-dontaudit vendor_init self:capability { sys_module net_raw };
-dontaudit vendor_init vendor_file:system module_load;
+allow vendor_init self:capability { sys_module net_raw };
+allow vendor_init vendor_file:system module_load;
 dontaudit vendor_init debugfs_tracing_instances:dir write;
-dontaudit vendor_init mediaserver:process setsched;
-dontaudit vendor_init self:udp_socket create;
+allow vendor_init mediaserver:process setsched;
+allow vendor_init self:udp_socket create;
 dontaudit vendor_init coreu_data_file:dir create_dir_perms;
diff --git a/wlan/iwlwifi/wlan_intel_restore.te b/wlan/iwlwifi/wlan_intel_restore.te
index deaa2da..f658a91 100644
--- a/wlan/iwlwifi/wlan_intel_restore.te
+++ b/wlan/iwlwifi/wlan_intel_restore.te
@@ -14,3 +14,6 @@ allow wlan_intel_restore toolbox_exec:file rx_file_perms;

 allow wlan_intel_restore block_device:dir search;

+dontaudit hal_wifi_default kernel:system module_request;
+dontaudit hal_wifi_default self:capability sys_module;
+
--
2.29.2

